build.gradle

```diff
@@ -0,0 +1,345 @@
+  compile(group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.4') {
+    // TODO remove when auth-checker-lib upgrades its dependency to spring boot 2.0.x
+    exclude group: 'uk.gov.hmcts.reform', module: 'java-logging-spring'
+
+  }
+  compile group: 'javax.inject', name: 'javax.inject', version: '1'
+  compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.swagger2Version
+  compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.swagger2Version
+  compile group: 'io.springfox', name: 'springfox-spring-webmvc', version: versions.swagger2Version
+
+  compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.2.2.RELEASE'
+  compile('org.springframework.boot:spring-boot-starter-hateoas')
+  compile group: 'org.springframework', name: 'spring-tx', version: '5.2.6.RELEASE'
+
+  testCompile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.30'
+
+  testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: versions.junit
+  testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: versions.junit
+  testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: versions.junit
+
+  testCompile group: 'org.pitest', name: 'pitest', version: versions.pitest
+  testCompile 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.4.7'
+  testCompile 'org.codehaus.sonar-plugins:sonar-pitest-plugin:0.5'
+
+  testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot) {
+    exclude group: 'junit', module: 'junit'
+  }
+  testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '2.7.0'
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign', version: '2.2.2.RELEASE'
+
+  compile group: 'io.cucumber', name: 'cucumber-java', version: versions.cucumber
+  testCompile group: 'io.rest-assured', name: 'rest-assured', version: versions.rest_assured
+  testCompile group: 'io.cucumber', name: 'cucumber-junit', version: versions.cucumber
+  testCompile group: 'io.github.openfeign', name: 'feign-jackson', version: versions.feign_jackson
+
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
+  smokeTestCompile group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
+  smokeTestCompile group: 'io.rest-assured', name: 'rest-assured', version: versions.rest_assured
```

Comment:
Do we really need these dependencies? Only befta-fw dependency should be enough for smoke and functional tests.

Reply:
needs actioning @nitishsingla1234 , @aashishaggarwal123

Reply:
Removed the above mentioned dependencies.